app/services/revenue_forecast_service.rb

```diff
@@ -48,16 +49,16 @@ def project_forecast(project, period = nil)
     result
   end
 
-  def self.year_forecast(year)
-    new.year_forecast(year)
+  def self.year_forecast(year, market = nil)
+    new.year_forecast(year, market)
   end
 
-  def year_forecast(year)
+  def year_forecast(year, market = nil)
```

Comment:
do we need `market = nil` in here since we already have it on the class method? (line 52)

Reply:
We don't. I'll fix this